src/GraphControl/Control/Grapher.cpp

```diff
@@ -792,6 +792,7 @@ namespace GraphControl
 
                 if (needsRenderPass)
                 {
+                    IsKeepCurrentView = true;
```

Comment:
Weird behavior in this scenario:
1. Make sure auto fit is on
2. Graph y=2 in f(1)
3. Graph y=300 in f(2)
4. Hide f(2) then hide f(1)
5. Notice graph snaps back to default position
6. Re-enable f(2) and f(1), graph is no longer correctly auto fitted even though it is toggled on

Reply:
This issue was already existing before the button change.